Shopify/ruby-lsp
# Fix check for RuboCop availability

### Motivation

The unavailable RuboCop warning was appearing even when RuboCop was available.

### Implementation

The previous approach seems somewhat fragile since it depends on the order that other classes are loaded. I think trying to require the gem is a better approach.

### Automated Tests

n/a

### Manual Tests

* Open a repo which uses RuboCop, e.g. `Shopify/tapioca`. Add `ruby-lsp` to the Gemfile, pointing to this branch. Ensure that RuboCop formatting is applied when saving a file, and that no warning about a missing RuboCop is shown.
